taskana-adapter-spring-boot-test/src/main/java/config/SimpleSpringBootTestConfig.java

```diff
@@ -0,0 +1,33 @@
+
+/**
+package config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jdbc.datasource.DataSourceTransactionManager;
+import org.springframework.transaction.PlatformTransactionManager;
+
+import javax.sql.DataSource;
+
+
+@Configuration
+public class SimpleSpringBootTestConfig {
+
+
+    @Bean
+    @ConfigurationProperties(prefix = "spring.datasource")
+    DataSource dataSource (){
+
+        DataSource dataSource = DataSourceBuilder.create().build();
+
+        System.out.println("################################################################"+dataSource.toString());
+
```

Comment:
System.out may be fine for test purposes, but it should be avoided in production code ...

Reply:
this whole test project was not supposed to be commited anyways, so it is deleted now